_pages/production.md

```diff
@@ -70,18 +85,18 @@ Before you can request deployment, you need to create a new and separate applica
 
 -   ATO approved environment
 
--   Approved logo for your app
+-   [Approved agency logo](https://developers.login.gov/design-guidelines/#agency-logo-guidelines) for your app
```

Comment:
Since this is the same repo, we can use baseurl here instead of hardcoding the domain, that way the links work in preview builds
```suggestion
-   [Approved agency logo]({{site.baseurl}}/design-guidelines/#agency-logo-guidelines) for your app
```